slice/IceGrid/Admin.ice

```diff
@@ -75,7 +75,7 @@ struct AdapterInfo
     /// The id of the adapter.
     string id;
 
-    /// A dummy direct proxy that contains the adapter endpoints.
+    /// A dummy direct proxy that contains the adapter endpoints. This proxy is never null.
```

Comment:
I don't think we can make this statement for struct fields in general. The application can always put a null proxy in a struct field.

Then, if you look at the Database implementation, this field is occasionally null.

Such information would be more valuable on an operation that accepts or returns an AdapterInfo or AdapterInfoSeq. Like (if accurate) "the proxy field of the returned adapter info is never null".